gtsam/linear/ConjugateGradientSolver.h

```diff
@@ -24,20 +24,22 @@
 namespace gtsam {
 
 /**
- * parameters for the conjugate gradient method
+ * Parameters for the Conjugate Gradient method
  */
 class GTSAM_EXPORT ConjugateGradientParameters : public IterativeOptimizationParameters {
 
-public:
+ public:
   typedef IterativeOptimizationParameters Base;
   typedef std::shared_ptr<ConjugateGradientParameters> shared_ptr;
 
+ protected:
```

Comment:
Hmmm. If anything, I’d make this a struct and remove the underscore so all fields can just be set and read easily. And deprecate the getters/setters, esp since we can now support this in python.